docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -311,13 +311,54 @@ Returns language translation keys as JSON string.
 
 ### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)
 
-Returns sized image URL from store `/content` directory.
+Returns URL for an image [uploaded to `/dav/content/`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).
 
-[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=getContentImage).
+#### Parameters
+
+* `a` **{String}:** Image path relative to `/dav/content/`
+* `width` **{Int}:** Width in pixels

```

Comment:
Might be worth mentioning that both width and height are optional.

Validation combinations are:
- No height/No width (returns original image)
- Width only
- Width + height

Not valid:
- Height only

This is true for both non-srcset helpers.

Reply:
Never mind, I see this is covered a bit later on.